server/segment_rpc.go

```diff
@@ -481,6 +481,7 @@ func SubmitSegment(ctx context.Context, sess *BroadcastSession, seg *stream.HLSS
 	}
 	if params.SegUploadTimeoutMultiplier > 1 {
 		uploadTimeout = time.Duration(params.SegUploadTimeoutMultiplier) * uploadTimeout
+		httpTimeout = time.Duration(params.SegUploadTimeoutMultiplier) * httpTimeout
```

Comment:
Would be good to take a pass at clarifying usage here sine SegUploadTimeoutMultiplier sounds like it only applies to upload, but now it also applies to the overall HTTP timeout for a full request.

Reply:
Addressed in https://github.com/livepeer/go-livepeer/pull/2667